rasa/__main__.py

```diff
@@ -64,7 +66,21 @@ def create_argument_parser() -> argparse.ArgumentParser:
 
 
 def print_version() -> None:
-    print("Rasa", version.__version__)
+    """Prints version information of rasa tooling and python."""
+    py_info, os_info = sys.version.split("\n")
+    rasa_x_info = None
+    try:
+        from rasax.community.version import __version__  # pytype: disable=import-error
+
+        rasa_x_info = __version__
+    except ModuleNotFoundError:
+        pass
```

Comment:
Does this work? 
```suggestion

    try:
        from rasax.community.version import __version__  # pytype: disable=import-error
        rasa_x_info = __version__
    except ModuleNotFoundError:
        rasa_x_info = None
```

Reply:
That extra line is caused by `black`. After import you need to have a seperating line.

Reply:
Sorry, i didn't mean the extra line, but rather not declaring `rasa_x_info = None` off the bat, handling that in the `except` instead.

Reply:
Ahhh like that! Sure, there's merit in that. Making the change.